src/ui/zcl_abapgit_html_toolbar.clas.abap

```diff
@@ -212,29 +219,37 @@ CLASS ZCL_ABAPGIT_HTML_TOOLBAR IMPLEMENTATION.
         ENDIF.
       ENDIF.
 
+
       IF <ls_item>-cur = abap_true.
-        lv_class = ' class="current-menu-item"'.
+        lv_class_value =  |current-menu-item { <ls_item>-li_class }|.
```

Comment:
if `li_class` is initial, then the class value will contain an extra space.

Reply:
I will add an IF

Reply:
@mbtools I added the IF. So I assume that the other PR #5230 which only changes the buttons to the toolbar (without the new one) can be merged now?
![image](https://user-images.githubusercontent.com/59128840/150190124-4780edca-7988-4648-abf1-46b8d16f7a8a.png)